server/db/schema.ts

```diff
@@ -1,25 +1,7 @@
 import { blob, integer, sqliteTable, text } from "drizzle-orm/sqlite-core";
 import { generateIdFromEntropySize } from "lucia";
 
-// Here we define our database schema as code
-// https://orm.drizzle.team/docs/column-types/sqlite
-
-/*
-TEMPLATE FOR NEW TABLES
-export const table = sqliteTable("table", {
-  id: text("id")
-    .primaryKey()
-    .$defaultFn(() => generateIdFromEntropySize(10)),
-  created_at: integer("created_at", { mode: "timestamp" })
-    .notNull()
-    .$defaultFn(() => new Date()),
-  updated_at: integer("updated_at", { mode: "timestamp" })
-    .notNull()
-    .$onUpdateFn(() => new Date()),
-});
-*/
-
-// temporary
+// Todos table
```

Comment:
its fine either way but usually with drizzle you still use camel case for the javascript table keys because it matches with the rest of js conventions

What actually gets used in the database is the string in the type function